core-web/libs/block-editor/src/lib/shared/components/suggestions/suggestions.component.ts

```diff
@@ -16,14 +14,10 @@ import { DotCMSContentlet, DotCMSContentType } from '@dotcms/dotcms-models';
 import { map, take } from 'rxjs/operators';
 import { MenuItem } from 'primeng/api';
 
-import { SuggestionsService, DotLanguageService } from '../../services';
-
-import {
-    SuggestionListComponent,
-    DEFAULT_LANG_ID,
-    suggestionOptions,
-    Languages
-} from '@dotcms/block-editor';
+import { DotLanguageService, Languages } from '../../services/dot-language/dot-language.service';
+import { SuggestionListComponent } from '../suggestion-list/suggestion-list.component';
+import { SuggestionsService } from '../../services/suggestions/suggestions.service';
+import { DEFAULT_LANG_ID, suggestionOptions } from '@dotcms/block-editor';
```

Comment:
We can add paths to the `tsconfig` to avoid `../../../` cuz it could get out of control fast.